src/acsets/schemas/catlab/SummationDecapode.json

```diff
@@ -0,0 +1,213 @@
+
+{
+    "version": {
+        "ACSets": "0.0.0",
```

Comment:
```suggestion
        "Catlab": "0.14.12",
```

Reply:
I actually generated this with ACSets v0.0.0 so I think this is right, but is this version supposed to be a minimum compatible Catlab version?

Reply:
The Catlab schema in this repo requires it to be catlab... not really sure I can comment on why it's this way or not. Good question for @olynch